introduction.md

```diff
@@ -1 +1,4 @@
 My name is Maciek and my favourite Code School path is JavaScript! 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?